src/components/FormDatePicker.vue

```diff
@@ -1,109 +1,109 @@
 <template>
-  <div class="form-group position-relative">
-    <label v-uni-for="name">{{label}}</label>
-    <date-picker
-      :config="config"
-      :value="date"
-      @input="setDate"
-      :disabled="disabled"
-      :placeholder="placeholder"
-      :data-test="dataTest"
-    />
-    <div v-if="(validator && validator.errorCount) || error" class="invalid-feedback d-block">
-        <div v-for="(error, index) in validator.errors.get(this.name)" :key="index">{{error}}</div>
-        <div v-if="error">{{error}}</div>
+    <div class="form-group position-relative">
+        <label v-uni-for="name">{{label}}</label>
+        <date-picker
+              :config="config"
+              :value="date"
+              @input="setDate"
+              :disabled="disabled"
+              :placeholder="placeholder"
+              :data-test="dataTest"
+        />
+        <div v-if="(validator && validator.errorCount) || error" class="invalid-feedback d-block">
+            <div v-for="(error, index) in validator.errors.get(this.name)" :key="index">{{error}}</div>
+            <div v-if="error">{{error}}</div>
+        </div>
         <small v-if="helper" class="form-text text-muted">{{helper}}</small>
```

Comment:
This is the line that has changed: everything else is PHPStorm applying formatting. Previously this was only displayed if there was error text.